internal/exec/stages/files/filesystemEntries.go

```diff
@@ -274,7 +287,10 @@ func (s stage) createEntries(fs types.Filesystem, files []filesystemEntry) error
 		if err := e.create(s.Logger, u); err != nil {
 			return err
 		}
+		// only relabel things on the root filesystem
+		if mnt == "/sysroot" {
```

Comment:
maybe check if filesystem is root instead of the mountpoint since I think `/sysroot` is an implementation detail.